src/cli/cli.ts

```diff
@@ -987,6 +988,21 @@ makeCommand<
   },
 });
 
+makeCommand({
+  name: 'write-action',
+  description: 'Send a write action to an AO Process',
+  options: [...writeActionOptions, optionMap.processId],
+  action: async (options) => {
+    const process = new AOProcess({
+      processId: requiredProcessIdFromOptions(options),
+    });
+    return process.send<AoMessageResult>({
+      tags: writeActionTagsFromOptions(options).tags ?? [],
+      signer: requiredAoSignerFromOptions(options),
+    });
+  },
+});
```

Comment:
adds some more utility to the CLI, being able to run custom write actions like 

```sh
node lib/esm/cli/cli.js write-action --process-id KF4wSJa5no2yyDjrDE3hxd5sJoGhLh0UEj8a6Rz3aX8 --wallet-file ../path/to/wallet.json --tags Action Register Registration-Type stake
```

Reply:
nit: name could be more aligned with SDK and require `action`, like `send-action` with `--action` option. same utility could be made for `read-action`

Reply:
fwiw ao.link uses `read` and `write`. that might be more user friendly than `send` -- and both methods will actually "send" and trigger some computation